docs/workshop/git_setup.md

```diff
@@ -1,12 +1,12 @@
 # Set up Github environment
 
-## Create Github account, Fork Repository, Generate Github Access token
+In order to get started with the workshop, it's vital to have a github account, so you can also interact with the codebase and make changes as and when required. Here are a few steps that you need to follow along to be all set to get started with this workshop.
 
-* **Create a Github account**: Go to[ github.com](https://github.com/) and click on sign up to create a new account.
+* **Create a Github account**: Go to[ github.com](https://github.com/) and click on `Sign Up` to create a new account.
 
```

Comment:
All images in this doc look quite unclear/low quality. Can these be updated?

Reply:
Let me add them again.